test/example/example_suite_test.go

```diff
@@ -14,7 +14,7 @@ import (
 
 var (
 	testenvInstance *testenv.TestEnv
-	testSuiteName   = "example-suite-" + testenv.RandomDNSName(6)
+	testSuiteName   = "example-" + testenv.RandomDNSName(3)
```

Comment:
why change from 6 to 3?

Reply:
Ginkgo throws an error total length kubernetes artifacts go above 24 chars. This wasn't a problem before as we were simply skipping this suite. Now we are initializing and skipping hence an error is thrown. I am also inclining towards removing this suite all together as it serves no purpose.